README.md

```diff
@@ -139,76 +169,136 @@ Once you've done that, restart the IDE.
 
 * [Access swagger api](http://localhost:8080/swagger-ui/index.html) and
   corresponding [openAPI docs here](http://localhost:8080/api-docs)
-    
+
 ## Quality Checks
 
 ### Jacoco
 
-* Generate Test reports and open [coverage report](build/reports/jacoco/test/html/index.html) 
+* Generate Test reports and open [coverage report](build/reports/jacoco/test/html/index.html)
+
 ```shell
 ./gradlew test jacocoTestReport
 ```
 
 * Check coverage minimum of 70%
+
 ```shell
 ./gradlew clean test jacocoTestCoverageVerification
 ```
 
-
 ### PMD
 
 * Run pmd for src
+
 ```shell
 ./gradlew pmdMain
 ```
 
 * Run pmd for test
+
 ```shell
 ./gradlew pmdTest
 ```
 
 ### SONAR
 
 #### Install sonarqube docker image locally
-  - Make sure you have docker installed on your machine - 
-    Download the installer using the url https://docs.docker.com/get-docker/. ( Prefer Docker Desktop Application )
-  - Start the docker application. Double-click Docker.app to start Docker.
+
+- Make sure you have docker installed on your machine -
+  Download the installer using the url https://docs.docker.com/get-docker/. ( Prefer Docker Desktop
+  Application )
+- Start the docker application. Double-click Docker.app to start Docker.
 
 Get the “SonarQube” image using the command
+
 ```shell
 docker pull sonarqube
 ```
+
 Start the "SonarQube" instance
+
 ```shell
 docker run -d --name sonarqube -e SONAR_ES_BOOTSTRAP_CHECKS_DISABLE=true -p 9000:9000 sonarqube:latest
 ```
 
 * Access SonarQube dashboard - http://localhost:9000</br>
   (default credentials )
-login: admin
-password: admin
+  login: admin
+  password: admin
 
 Ref: https://docs.sonarsource.com/sonarqube/latest/try-out-sonarqube/
 
-#### Set-up wcc-backend project on local sonarQube instance 
-#### Step 1
+#### Set-up wcc-backend project on local sonarQube instance
+
+* Step 1
+
 1. Select create a local project
 2. "Project display name" = wcc-backend
 3. "Project key" = wcc-backend
 4. "Main branch name" = *
-#### Step 2
+
+* Step 2
+
 1. Choose the baseline for new code for this project</br>
-select "Use the global setting"
+   select "Use the global setting"
 2. Click "Create Project" at the bottom
 
-#### Step 3
+* Step 3
+
 Generate token to replace in the project.
+
 1. Click "Locally" on the main dashboard
 2. Generate a token on the next screen ( choose Expires in - No expiration) [ Click Generate]
 3. Copy the token = "sqp_XXXXXXX" and replace in the file <b> build.gradle.kts </b><br>
    <b>property("sonar.token", "PLACE_YOUR_TOKEN_HERE")</b>
 
 #### Perform SONAR ANALYSIS
+
 ```shell
 ./gradlew sonarQubeAnalysis -PlocalProfile
-```
\ No newline at end of file
+```
+
+## Deploy application
+
+### Deploy with docker
+
+* build create jar: `./gradlew clean bootJar`
+
+* start app via docker
+
+```shell
+docker build -t wcc-backend .
+docker run -d -p 8080:8080 --name wcc-backend-container wcc-backend
+```
+
+* start app via docker with remote debug
```

Comment:
what is difference between this command running docker in debug with the next one ?

Reply:
sorry I am not sure what are you mentioning, are you talking about these two: 

`docker run -d -p 8080:8080 --name wcc-backend-container wcc-backend`

and this:
`docker run -e "JAVA_TOOL_OPTIONS=-agentlib:jdwp=transport=dt_socket,address=5005,server=y,suspend=n" -p 8080:8080 -p 5005:5005 -t wcc-backend`

?

Reply:
sorry it is duplicated.